network/iptables.go

```diff
@@ -135,7 +173,7 @@ func ipTablesRulesExist(ipt IPTables, rules []IPTablesRule) (bool, error) {
 }
 
 // ipTablesCleanAndBuild create from a list of iptables rules a transaction (as string) for iptables-restore for ordering the rules that effectively running
-func ipTablesCleanAndBuild(ipt IPTables, rules []IPTablesRule) (IPTablesRestoreRules, error) {
+func ipTablesCleanAndBuild(ipt IPTables, rules []IPTablesRule, toChain bool) (IPTablesRestoreRules, error) {
```

Comment:
Can you explain in the function comment what "toChain" mean?

Reply:
If the rule is used to send the traffic to the chain and it has to be added with -I and not -A.
Maybe I can modified to a more meaningful name.